src/constants/inventory.js

```diff
@@ -328,4 +328,87 @@ export let inventory = [
     },
 ];
 
+// Opdracht 1a
+
+const mapMethod = inventory.map((tvType) => {
+    return tvType.type;
+});
+console.log(mapMethod);
+
+console.log("---------------");
+
+
+// Opdracht 1b
+
+const filterMethod = inventory.filter((soldTv) => {
+    return soldTv.originalStock === soldTv.sold;
+});
+console.log(filterMethod);
+
+console.log("---------------");
+
+
+// Opdracht 1c
+
+const findMethod = inventory.find((smartTv) => {
+    return smartTv.type === "NH3216SMART";
+});
+console.log(findMethod);
+
+console.log("---------------");
+
+
+// Opdracht 1d
+
+const suiteableTvs = inventory.map((tv) => {
+    return "name: " + tv.brand + tv.name + ", suitable: " + (tv.refreshRate >= 100);
```

Comment:
Ik vind het volgende altijd iets beter te lezen
```suggestion
    return `name: ${tv.brand} ${tv.name}, suitable: ${tv.refreshRate >= 100}`;
```